### DISCUSSION TITLE
Setup SSR with Nginx

### DISCUSSION THREAD

**Discussion by romsar** [inertiajs/inertia#1041] [Help] [answered]
My app is running via NGINX.
Am I able to setup ssr with nginx or I should completely disable nginx and only run node process?
By the way, should I add command "node ssr.js" to supervisord?

**Comment by ajnsn** [comment, marked as answer]
Hey @RomanSarvarov 
I am not 100% sure if I understand your question correctly. Generally speaking your webserver (Nginx) is still responsible for delivering all content to your users. The node process is only called internally (e.g. http://127.0.0.1:13714/render) and only delivers the rendered content for the first visit. All subsequent Inertia visits will be XHR visits.

Hope this helps :)
